testing/base_installation_for_test.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
```

Comment:
@KyleBehiels If we can we can we should just use the standard install script. Therefore we don't have to go back and change both install scripts one after the other. For example the pip installs below need to be changed in both files. **If their is specific install commands we need to skip for testing we can use a shell argument and some if statements to skip certain steps in the install process.** Its could to test the base install in your CI system as it makes sure that the install will work every time.